ci-operator/jobs/openshift/odo/openshift-odo-master-presubmits.yaml

```diff
@@ -26,12 +26,12 @@ presubmits:
         - name: CONFIG_SPEC
           valueFrom:
             configMapKeyRef:
-              key: redhat-developer-odo-master.yaml
+              key: openshift-odo-master.yaml
               name: ci-operator-master-configs
         - name: JOB_NAME_SAFE
           value: e2e-core
         - name: TEST_COMMAND
-          value: |-
+          value: |
```

Comment:
interesting

Reply:
Yes it is. I wondered what happened there as these files are generated and not edited by hand